test/half.h

```diff
@@ -262,7 +262,7 @@ struct half_t
 
     /// numeric_traits<half_t>::max
     __host__ __device__ __forceinline__
-    static half_t max() {
+    static half_t (max)() {
```

Comment:
👍 I agree that it'd be best to just have one approach, but this can be done as a follow up. We'd need to deprecate this macro (and the equivalent one in Thrust) if we do this, but it's very low priority right now.